lib/classes/Service.js

```diff
@@ -93,6 +93,7 @@ class Service {
         that.plugins = serverlessYml.plugins;
         that.resources = serverlessYml.resources;
         that.functions = serverlessYml.functions;
+        that.runtime = serverlessYml.runtime;
```

Comment:
Agreed with @pmuens ... and even if it wasn't provider specific, `runtime` isn't really a "variable" imo. Its easy to get the `runtime` from `serverless.yml` anytime, and the same goes for `provider` and `service` properties. I think the **real** variables are `stage` & `region`.

Do you think there may be a need for these extra "variables"?

Thanks for pitching in on this! ☺️